front/src/app/todo-list/todo-list.component.scss

```diff
@@ -0,0 +1,31 @@
+.todo-lists ::ng-deep {
+  mat-radio-group {
+    display: block;
+  }
+  mat-radio-button {
+    width: 220px;
```

Comment:
1. Add an empty line between selectors of the same level
2. Better use class names than component selectors in css